core/feconf.py

```
@@ -698,14 +684,16 @@ def get_empty_ratings() -> Dict[str, int]:
 MODERATOR_ACTION_UNPUBLISH_EXPLORATION = 'unpublish_exploration'
-DEFAULT_SALUTATION_HTML_FN: Callable[[str], str] = (
-    lambda recipient_username: 'Hi %s,' % recipient_username
-)
-DEFAULT_SIGNOFF_HTML_FN: Callable[[str], str] = lambda sender_username: (
-    'Thanks!<br>%s (Oppia moderator)' % sender_username
-)
-DEFAULT_EMAIL_SUBJECT_FN: Callable[[str], str] = lambda exp_title: (
-    'Your Oppia exploration "%s" has been unpublished' % exp_title
+DEFAULT_SALUTATION_HTML_FN: Callable[  # pylint: disable=invalid-name
+    [str], str
+] = lambda recipient_username: f'Hi {recipient_username},'
```

Ruff (0.15.21)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `DEFAULT_SALUTATION_HTML_FN` as a `def`

(E731)